llvm_util/llvm2alive.cpp

```diff
@@ -256,11 +256,20 @@ class llvm2alive_ : public llvm::InstVisitor<llvm2alive_, unique_ptr<Instr>> {
     string fn_name = '@' + fn->getName().str();
     auto call = make_unique<FnCall>(*ty, value_name(i), move(fn_name), flags,
                                     !known);
-    for (auto &arg : i.args()) {
```

Comment:
it seems the iterator could be kept as before. There's no benefit in this change AFAICT